### PR TITLE
Support Task resource

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Support Task resource

### Check List
- [ ] New functionality includes sanity testing.
  - [ ] All sanity tests pass. (`$ ansible-test sanity`).
  - [ ] All unit tests pass.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ This release extends the planned support of the collections to OneView REST API
 - oneview_storage_volume_attachment_facts
 - oneview_storage_volume_template
 - oneview_storage_volume_template_facts
+- oneview_task
```
Comment (AsisBagga) [inline CHANGELOG.md:93]: if new endpoints are added for the task resource, then please mention them separately like below:

Added support for a new endpoint in oneview_logical_interconnect.
POST /rest/logical-interconnects/bulk-inconsistency-validation

File: plugins/modules/oneview_task_facts.py [hunk 2]
```diff
@@ -52,7 +53,7 @@
     hostname: 172.16.101.48
```
Comment (AsisBagga) [inline plugins/modules/oneview_task_facts.py:53]: please remove the IP address.

File: roles/oneview_task/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## License
+
+Apache
+
+#### Return Values
+
+| Name          | Description  | Returned | Type       |
+| ------------- |-------------| ---------|----------- |
+| task | Has the facts about the managed OneView Tasks |  On state 'collect'. Can be null. |  dict |
```
Comment (AsisBagga) [inline roles/oneview_task/README.md:36]: change task to tasks

Comment (yuvirani) [reply roles/oneview_task/README.md]: Done

File: roles/oneview_task/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+## oneview_task
+Manage OneView Task resources.
+
+#### Synopsis
+ Provides an interface to manage Task resources. Can update.
+
+#### Requirements (on the host that executes the module)
+  * hpeOneView >= 6.1.0
```
Comment (SHANDCRUZ) [inline roles/oneview_task/README.md:8]: python version is missing.

File: roles/oneview_task/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+---
+# defaults file for oneview_task
+config: "~/.ansible/collections/ansible_collections/hpe/oneview/roles/oneview_task/files/oneview_config.json"
```
Comment (SHANDCRUZ) [inline roles/oneview_task/defaults/main.yml:3]: contents variable is missing.

**Review by SHANDCRUZ — APPROVED** [review]

**Review by VenkateshRavula — COMMENTED** [review]
update tasks resource in https://github.com/HewlettPackard/oneview-ansible-collection/blob/master/oneview-ansible-collection.md

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**